lib/createWorkers.js

```diff
@@ -48,10 +50,37 @@ function createWorkers(handlers, config, options = {}) {
 
   return {
     listen: co.wrap(_listen),
-    close: co.wrap(_close)
+    close: co.wrap(_close),
+    get emitter() {
+      return emitter;
+    },
+    wait(key) {
+      return new Promise(resolve => emitter.once(key, resolve));
```

Comment:
I would have used wait(event) or wait(eventName) for clarity, but that is fine...